README.md

```diff
@@ -70,16 +70,16 @@
 python3 manage.py createsuperuser --username admin
 ```
 
-**Setup is now complete!** :tada:
+**Setup is now complete!** :tadaa:
 
-## Launch
-First, make sure that the virtual environment we created, i.e. `ve`, is activated. Then run
+## Launch 
+First, make sure that the virtual environment we created, i.e. `ve`, is now activated. Then run
```

Comment:
I think "is activated" would be the right phrase here, instead of "is now activated".
There's also a trailing space after the heading "Launch" (seen in bright green) which should be removed.

Reply:
too particular about it... 👍 :-)